superset/utils/pandas_postprocessing.py

```diff
@@ -357,8 +359,10 @@ def rolling(  # pylint: disable=too-many-arguments
     rolling_type_options = rolling_type_options or {}
     df_rolling = df[columns.keys()]
     kwargs: Dict[str, Union[str, int]] = {}
-    if not window:
+    if not window and window != 0:
```

Comment:
I believe this is equivalent? Unless there is some python silliness I'm not familiar with.

```suggestion
    if window is None:
```

Reply:
nice catch! I will change type of `window`